tests/instances.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { test, Page } from "@playwright/test";
+import { TEXT_TIMEOUT } from "./constants";
+
+test("instance create and remove", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await deleteInstance(page, instance);
+});
+
+test("snapshot create restore and remove ", async ({ page }) => {
+  const instance = randomInstanceName();
+  const snapshot = randomSnapshotName();
+  await createInstance(page, instance);
+  await openInstanceDetail(page, instance);
```

Comment:
See the other comments: with this, we are opening the instance detail twice: first by click (in `createInstance`) and then by navigation (in `openInstanceDetail`).